### ISSUE TITLE
Sample laravel model, controller, api, crud operator

### ISSUE THREAD

**Issue by codebangla** [moell-peng/mojito#11]
Hello,

First of all thanks a lot for your awesome package. Would you please create a sample SCRUD using Laravel model, controller like SCRUD for example, employee or todo?

I am not sure how are you accessing the API using postman or insomnia? Where are you getting your laravel backend API? I've found the js/api directory where you have the endpoints. But are you generating those endpoints from the Spatie package?

Thanks

**Comment by moell-peng** [comment]
@codebangla 
The API route is registered here:
https://github.com/moell-peng/mojito/blob/master/src/Providers/MojitoServiceProvider.php

Https://github.com/moell-peng/mojito/blob/master/src/routes.php

For CURD examples you can refer to the source code, for example:
https://github.com/moell-peng/mojito/blob/master/src/Http/Controllers/PermissionController.php

**Comment by codebangla** [comment]
@moell-peng Yes I already found them using route:list, thanks

I found an issue. The pages do not refresh the state automatically whenever I add something using the modal form, for example, the permission group. 

The UX is like this for me, 
I add using the modal/popup form, the green toast shows and fades then view remains the same, I refresh the browser, then I see the updated view I mean, I can see the newly added record.
